src/main.rs

```diff
@@ -355,6 +355,7 @@ impl Future for Main {
 }
 
 fn main() {
+    env::set_var("RUST_BACKTRACE", "full");
```

Comment:
Can you make this a default only:
```rust
if env::var("RUST_BACKTRACE").is_none() {
    env::set_var("RUST_BACKTRACE", "full");
}
```

Reply:
Done.

Reply:
Had to change it to:
```
match env::var_os("RUST_BACKTRACE") {
        None => env::set_var("RUST_BACKTRACE", "full"),
        Some(_) => {}
    }
```
as ```env::var``` has no method ```is_none``` apparently.

Reply:
It returns a `Result` rather than an `Option` so it'd be `is_err()` (if you still want the more concise original)

Reply:
changed accordingly.